config-schema.json

```diff
@@ -341,18 +352,21 @@
                       "$id": "#/properties/entities/properties/remote/items/properties/commands/items/properties/code",
                       "type": "string",
                       "title": "The Code Schema",
+                      "default": null,
                       "examples": [
                         "0000,006C,0000,0022,00AD,00AD,0016,0041,0016,0041,0016,0041,0016,0016,0016,0016,0016,0016,0016,0016,0016,0016,0016,0041,0016,0041,0016,0041,0016,0016,0016,0016,0016,0016,0016,0016,0016,0016,0016,0016,0016,0041,0016,0016,0016,0016,0016,0016,0016,0016,0016,0016,0016,0016,0016,0041,0016,0016,0016,0041,0016,0041,0016,0041,0016,0041,0016,0041,0016,0041,0016,06FB"
                       ],
-                      "pattern": "^(.*)$"
+                      "pattern": ""
```

Comment:
Here a pattern makes sense and should be defined.
An IR code must be in a defined format, otherwise it cannot be processed.